pkg/master/master.go

```diff
@@ -182,8 +182,10 @@ type Config struct {
 	// allow downstream consumers to disable swagger
 	EnableSwaggerSupport bool
 	// allow api versions to be conditionally disabled
-	DisableV1 bool
-	EnableExp bool
+	DisableV1                 bool
+	EnableExtensions          bool
+	EnableExtensionResources  map[string]struct{} // If present, only these resources are enabled in the extensions api group, all others are disabled. This is ignored if EnableExtensions is false.
```

Comment:
Note that only one of enableExtensionResources or disableExtensionResources can be non-empty.
By keeping them separate, checking that is easy (len(enableExtensionResources) == 0 || len(disableExtensionResources) == 0). Else, we will have to iterate through all elements and then see that all of them have the same bool value.